scala/sbt/dse/src/main/scala/com/datastax/spark/example/WriteRead.scala

```diff
@@ -20,29 +18,28 @@ object Example extends App {
     session.execute("""CREATE TABLE IF NOT EXISTS ks.kv (k int, v int, PRIMARY KEY (k))""")
   }
 
-
   // A SparkContext
   val sc = new SparkContext(conf)
-
-  // This Hive Context will use the DSE HiveMetaStore
-  val sqlContext = new HiveContext(sc)
+  val sqlContext = new CassandraSQLContext(sc)
```

Comment:
Several Reasons we Don't Want Anyone using CSC

1) Limited featureset
2) No longer really supported by spark
3) HiveContext is better in every way